lib/Api.js

```diff
@@ -140,6 +140,11 @@ class Api {
 
     self.db.getProjectDiskTotal(projectId)
       .then(function(result) {
+        // Zero is returned as an empty object rather than a proper number if
+        // we don't convert it to a string.   :-(
```

Comment:
Haha! Yeah. Zero is always a problem for loosely typed languages. It drives me bonkers.